TeamH Wiki Update - Traffic Modelling Datasets

HobbySingh opened the pull request.

smbryan commented on wiki/datasets/traffic-modelling-datasets.md:
```diff
@@ -0,0 +1,16 @@
+
+Each dataset is scored based upon several parameters crucial for learning behavior and interactions between vehicles in a recorded scenario.
+
+<iframe height="100%" width="100%" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vRNtQVog9cMhYrOuv0fGlEKyK4HiuwD0orLoXEfO9z-Jtz9hvEB6qzdZy6lOSsyyPGakYe6ggYBAi8u/pubhtml?gid=0&amp;single=true&amp;widget=true&amp;headers=false"></iframe>
```
This spreadsheet should be here and not linked to a Google doc. The spreadsheet also appears to be incomplete.

HobbySingh commented:
> This spreadsheet should be here and not linked to a google doc. The spreadsheet also appears to be incomplete.

The table is broader and it is not possible to format that using Markdown. Hence I put that into a separate spreadsheet as it looks much better visually.

I have fixed the incomplete part.

smbryan commented:
I applaud your ingenuity, but I can't accept adding that dependency. Markdown works just fine: https://kramdown.gettalong.org/quickref.html#tables

HobbySingh commented:
I have committed the file that I already prepared with a table embedded in Markdown. You can look that the Markdown isn't flexible with large tables.

HobbySingh commented:
Hi, I have updated the file. 

1. Reduced the table content so that it fits nicely in markdown and moved it as general description.
2. Added another dataset.

Please let me know if other changes are required.

smbryan commented:
Looks good. Thanks!
